lib/skywalking/span.lua

```diff
@@ -102,11 +102,7 @@ function _M.createExitSpan(operationName, context, parent, peer, contextCarrier)
 
         local firstSpan = context.internal.first_span
         local parentEndpointName
-        if context.internal.first_ref then
-            parentEndpointName = context.internal.first_ref.entry_endpoint_name
-        else
-            parentEndpointName = firstSpan.operation_name
-        end
+        parentEndpointName = firstSpan.operation_name
```

Comment:
Do you remove `first_ref` variable? I think it is useless today. @mrproliu

Reply:
That's true, I will make a new PR to remove this variable.